core/src/main/java/io/bisq/core/dao/compensation/CompensationRequestManager.java

```diff
@@ -141,15 +141,14 @@ public void addToP2PNetwork(CompensationRequestPayload compensationRequestPayloa
     }
 
     public CompensationRequest prepareCompensationRequest(CompensationRequestPayload compensationRequestPayload)
-            throws InsufficientMoneyException, ChangeBelowDustException, TransactionVerificationException, WalletException, IOException {
+            throws InsufficientMoneyException, ChangeBelowDustException, TransactionVerificationException, WalletException, IOException, CompensationAmountException {
+        if (compensationRequestPayload.getRequestedBsq().compareTo(feeService.getCreateCompensationRequestFee()) <= 0) {
+            throw new CompensationAmountException(feeService.getCreateCompensationRequestFee(), compensationRequestPayload.getRequestedBsq());
```

Comment:
I think that is not correct. The requested BSQ are not the BSQ used for the fee payment and it would be a < not a <= comparison.

Reply:
As discussed on slack: if the minimum compensation amount is less than or equal to the bsq fee then issuance output will be treated as a bsq output. Example with bsq fee = 1 BSQ and issuance request for 1 BSQ
Input 0: 100 BSQ (0.001 BTC)
Input 1: 0.00001 BTC 
Output 0: 99 BSQ (0.00099 BTC)
Output 1: issuance of 1 BSQ (0.00001 BTC)

Output 1 is supposed to be the issuance output using the BTC from input 1 but will actually be treated as a BSQ output using the remaining 1 BSQ from input 0. No BSQ will be burnt and it would create a malformed compensation request. By requiring the issuance to be more than the BSQ fee (which makes sense anyway, who would want to pay more than the earn to issue new BSQ) this is no longer an issue.

Reply:
I will introduce a minCompRequestAmount to cover that.